### PR TITLE
Make Multiversion opt-in

### PR DESCRIPTION
**Pull request by jeqq**

### CODE REVIEW

File: src/MultiversionManager.php [hunk 1]
```diff
@@ -73,32 +73,6 @@ class MultiversionManager implements MultiversionManagerInterface, ContainerAwar
...
-  protected $entityTypeBlackList = array(
-    'user',
-    'shortcut',
-    'contact_message',
-    'content_moderation_state',
-    'entity_test_no_id',
-    'entity_test_base_field_display',
-  );
-
-  /**
```
Comment (dickolsson) [inline src/MultiversionManager.php]: Nice to get rid off this!

File: config/install/multiversion.settings.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - menu_link_content
+  - block_content
+  - file
+  - media
+  - shortcut
+  - entity_test
+  - entity_test_rev
+  - entity_test_mul
+  - entity_test_mulrev
+  - content_moderation_state
```
Comment (dickolsson) [inline config/install/multiversion.settings.yml]: @timmillwood Do you think we should replicate `content_moderation_state` by default?

Comment (jeqq) [inline config/install/multiversion.settings.yml]: @timmillwood @dickolsson I've already removed storage schema handler override.

**Comment by dickolsson** [comment]
@jeqq Is it possible to write tests for Drush commands? Do you think it would be worth doing that?

**Pull request merged**